src/scss/tools/functions/_units.scss

```diff
@@ -57,3 +57,18 @@
         @return $number;
     }
 }
+
+@function ko-px($emVal, $base: $font-size-base) {
```

Comment:
Would be good to rename these so that they are fozzie related (as eventually we're going to port the functions from the kickoff utils and rename them).

So maybe `em-to-px` and just drop the `ko-` from the function below so `map-to-px`